src/view/pages/Farmer/index.tsx

```diff
@@ -9,21 +9,8 @@ const createCollect = () => (
   <Collect />
 );
 

```

Comment:
I can't comment on line `5`, but remove `Buy` since we don't have that anymore.

Reply:
Removed, along with test data